tools/tests/merger/fixtures/extractor/opensearch/expected_1.3.yaml

```diff
@@ -95,42 +97,45 @@ components:
               - tagline
             unevaluatedProperties:
               type: object
-    info@500:
+      description: ''
+    info.500:
       content:
         application/json:
           schema:
             type: object
             properties:
               tagline:
                 type: string
-    info@503:
+      description: ''
+    info.503:
       content:
         application/json:
           schema:
             type: object
-    info@added-1.3-removed-2.0:
+      description: ''
+    info.added-1.3-removed-2.0:
       description: Added in 1.3, removed in 2.0 via attribute in response body.
       x-version-added: '1.3'
       x-version-removed: '2.0'
-    info@distributed-all:
+    info.distributed-all:
       description: Distributed in opensearch.org, AOS and AOSS.
-    info@removed-2.0:
+    info.removed-2.0:
       description: Removed in 2.0 via attribute next to ref.
-    info@removed-2.0-refs:
+    info.removed-2.0-refs:
       description: One of the ref values removed in 2.0.
       schema:
         oneOf:
-          - $ref: '#/components/schemas/_common:Type'
-          - $ref: '#/components/schemas/_common:OldId'
```

Comment:
Can we also replace `:` with `___` in this case to separate the schema category from the type name? Similar to the previous comment regarding the response naming, this can get confusing with more complex schema categories say `ai_dsl.aggregation_common_Type`. It's hard to visually tell where the category ends and where the schema name starts. `ai_dsl.aggreation_common___Type` is slightly better and easier for the CodeGen to parse.

Reply:
Take a look at the update, we have to replace a few things, I went with the following map.

```
      .replaceAll('::', '___')
      .replaceAll('@', '__')
      .replaceAll(':', '_')
```